sdk/src/androidTest/java/com/mapbox/maps/OfflineTest.kt

```diff
@@ -287,19 +287,17 @@ class OfflineTest {
     val latch = CountDownLatch(2)
     var resourceRequests = 0
     var mapLoadingErrorCount = 0
-    val observer = object : Observer() {
-      override fun notify(event: Event) {
-        Logger.e(TAG, "type ${event.type}, data ${event.data.toJson()}")
-        val data = event.getResourceEventData()
-        if (!data.cancelled && data.dataSource == "database" && data.request.kind == "tile") {
-          resourceRequests++
-        }
-        if (event.type == MapEvents.MAP_LOADING_ERROR) {
-          mapLoadingErrorCount++
-        }
-        if (event.type == MapEvents.MAP_LOADED || event.type == MapEvents.STYLE_LOADED) {
-          latch.countDown()
-        }
+    val observer = Observer { event ->
+      Logger.e(TAG, "type ${event.type}, data ${event.data.toJson()}")
+      val data = event.getResourceEventData()
+      if (!data.cancelled && data.dataSource == "database" && data.request.kind == "tile") {
+        resourceRequests++
+      }
+      if (event.type == MapEvents.MAP_LOADING_ERROR) {
```

Comment:
```suggestion
      when (event.type) {
         MapEvents.MAP_LOADING_ERROR -> mapLoadingErrorCount++
         MapEvents.MAP_LOADED, MapEvents.STYLE_LOADED -> latch.countDown()
       }
```

Reply:
I'll not be doing refactoring existent code as part of this PR, this PR is to adopt to latest core only.
We can revisit those nit's later as part of some other cleanup.